src/tracks/queue.rs

```diff
@@ -278,6 +278,13 @@ impl TrackQueue {
 
         inner.stop_current()
     }
+
+    /// Returns Vec of all handles in queue
```

Comment:
Main points to mention in docs after the headline:
* Reordering the queue can't be done here, and has to go through `modify_queue`. The returned Vec is a snapshot of the queue at the time of call, and might change afterwards.
* Calling `stop` on a handle will effectively skip that track.

Minor nits:
* can you please end doc comments with a full stop?
* Maybe something like "Returns a list of currently queued tracks." for the headline?